foundry.toml

```
@@ -0,0 +1,15 @@
+[default]
+src = 'src'
+out = 'out'
+libs = ['lib']
+solc-version = "0.8.7"
+optimizer = true
```

```suggestion
solc-version = "0.8.7"
```
Most of these entries are just copied forge defaults and they don't improve readability by being explicit, IMO we could drop them for cleanness.

---

Okay, I will remove them.

---

I removed all default entries and only added the ones that need to be overridden by the entries that are specified in the current `.dapprc`.